tests/unit/test_column_noise.py

```diff
@@ -643,16 +740,19 @@ def test_use_nickname(dummy_dataset):
             chosen_nicknames = noised_data.loc[
                 real_name_idx.difference(noised_data.index[noised_data == real_name])
             ]
-            chosen_nickname_weights = pd.Series(
-                chosen_nicknames.value_counts() / sum(chosen_nicknames.value_counts())
-            )
-            name_weight = 1 / len(names_list.loc[real_name])
-            # We are weighting our rtol to adjust for variance depending on number of nicknames
-            rtol = 0.025 * len(chosen_nickname_weights)
-            is_close_wrapper(chosen_nickname_weights, name_weight, rtol)
-
-
-def test_use_fake_name(dummy_dataset):
+            chosen_nickname_counts = pd.Series(chosen_nicknames.value_counts())
+            expected_name_proportion = 1 / len(names_list.loc[real_name])
+            for nickname in chosen_nickname_counts.index:
+                fuzzy_checker.fuzzy_assert_proportion(
+                    name="use_nickname_proportion",
+                    observed_numerator=chosen_nickname_counts.loc[nickname],
+                    observed_denominator=chosen_nickname_counts.sum(),
+                    target_proportion=expected_name_proportion,
+                    name_additional=" for nickname: " + nickname,
```

Comment:
```suggestion
                    name_additional=f"for nickname: {nickname} of real name: {real_name}",
```

Could be a bit clearer